tests/store/test_views.py

```diff
@@ -6,10 +6,11 @@
 
 
 class StoreViews(TestCase):
-    render_templates = False
+    render_templates = True
```

Comment:
It looks like the default value to this property is True so keeping this around isn't strictly required. Same as in the above file.....unless I'm wrong ;)